.github/workflows/ci.yml

```diff
@@ -5,8 +5,29 @@ on:
     branches:
       - main
   pull_request:
+  schedule:
+    # run CI every day even if no PRs/merges occur
+    - cron: '0 12 * * *'
 
 jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: cachix/install-nix-action@v17
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
+      - name: Format
+        run: nix-shell --pure --run "make format"
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: cachix/install-nix-action@v17
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
+      - name: Lint
+        run: nix-shell --pure --run "make lint"
```

Comment:
Out of curiosity: do these mutually cache, i.e. will running one speed up the other?